google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClient.java

```diff
@@ -775,6 +790,189 @@ public void awaitReplication(String tableId) {
         stub.awaitReplicationCallable().futureCall(tableName));
   }
 
+  @BetaApi("Managed Backup feature is in alpha and may change in the future.")
+  public Backup createBackup(CreateBackupRequest request) {
+    return ApiExceptions.callAndTranslateApiException(createBackupAsync(request));
+  }
+
+  @BetaApi("Managed Backup feature is in alpha and may change in the future.")
+  public ApiFuture<Backup> createBackupAsync(CreateBackupRequest request) {
+    return ApiFutures.transform(
+        stub.createBackupOperationCallable().futureCall(request.toProto(projectId, instanceId)),
+        new ApiFunction<com.google.bigtable.admin.v2.Backup, Backup>() {
+          @Override
+          public Backup apply(com.google.bigtable.admin.v2.Backup backupProto) {
+            return Backup.fromProto(backupProto);
+          }
+        },
+        MoreExecutors.directExecutor());
+  }
+
+  @BetaApi("Managed Backup feature is in alpha and may change in the future.")
+  public Backup getBackup(String clusterId, String backupId) {
+    return ApiExceptions.callAndTranslateApiException(getBackupAsync(clusterId, backupId));
+  }
+
+  @BetaApi("Managed Backup feature is in alpha and may change in the future.")
+  public ApiFuture<Backup> getBackupAsync(String clusterId, String backupId) {
+    GetBackupRequest request =
+        GetBackupRequest.newBuilder()
+            .setName(NameUtil.formatBackupName(projectId, instanceId, clusterId, backupId))
+            .build();
+    return ApiFutures.transform(
+        this.stub.getBackupCallable().futureCall(request),
+        new ApiFunction<com.google.bigtable.admin.v2.Backup, Backup>() {
+          @Override
+          public Backup apply(com.google.bigtable.admin.v2.Backup backup) {
+            return Backup.fromProto(backup);
+          }
+        },
+        MoreExecutors.directExecutor());
+  }
+
+  @BetaApi("Managed Backup feature is in alpha and may change in the future.")
+  public List<String> listBackups(String clusterId) {
+    return ApiExceptions.callAndTranslateApiException(listBackupsAsync(clusterId));
+  }
+
+  @BetaApi("Managed Backup feature is in alpha and may change in the future.")
+  public ApiFuture<List<String>> listBackupsAsync(String clusterId) {
```

Comment:
The python library adds additional parameters to allow for an order by in the request: https://github.com/googleapis/python-bigtable/pull/57/files#diff-2b7fa7c0cc3c25cbb236202352ee3ecdR731. Is this something we can add in here as well?

Reply:
That could be a nice enhancement. And order_by, filter and pagination could all be added to the list API. However, given the current Java client does not support them for other APIs, I feel it would be better to add them in a separate PR.

Reply:
SGTM